ReviewBot.py

```diff
@@ -209,11 +209,11 @@ def add_review(self, req, by_group=None, by_user=None, by_project=None, by_packa
 
         for r in req.reviews:
             if (r.by_group == by_group and
-                r.by_project == by_project and
-                r.by_package == by_package and
-                r.by_user == by_user and
-                # Only duplicate when allow_duplicate and state != new.
-                (not allow_duplicate or r.state == 'new')):
+                    r.by_project == by_project and
+                    r.by_package == by_package and
+                    r.by_user == by_user and
+                    # Only duplicate when allow_duplicate and state != new.
+                    (not allow_duplicate or r.state == 'new')):
```

Comment:
Hmm, this class of indent is useful since everything lines up vertically to scan rather than being offset from the first entry.